dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactory.java

```diff
@@ -51,7 +51,7 @@ default FieldFactory instance() {
     List<Field> selectByContentTypeInDb(String id) throws DotDataException;
 
 
-    String suggestVelocityVar(String tryVar, List<Field> takenFields) throws DotDataException;
+    String suggestVelocityVar(String tryVar, List<String> takenFields) throws DotDataException;
```

Comment:
doc, why are you changing this method signature?, if the List<String> is a list with the Field's id (or some other field attribute) I think is better keep the List of Fied and into the method get the field's id

Reply:
The only thing we used from Field was its name